### PR TITLE
refactor: simplifying code in human_annotators

### PR DESCRIPTION
**Pull request by IgorTest19**
Restructuring code and reducing complex conditions in human_annotators_common_errors

### CODE REVIEW

File: human_annotators_common_errors.py [hunk 1]
```diff
@@ -151,82 +116,100 @@ def get_answer(error_type: str, element_correct: str) -> str:
...
+                    'Usunięto błędy w stosowaniu interpunkcji, oto poprawiony tekst',
+                    'Tekst po usunięciu nieprawidłowej interpunkcji prezentuje się następująco',
+                    'Oto tekst po korekcie interpunkcyjnej'
+            ]
+    }
+
+    return f"{random.choice(answer_map.get(error_type))}:\n{element_correct}"
+
+
+def crate_dirs() -> None:
```
Comment (pawelkiszczak) [inline human_annotators_common_errors.py]: Is it a typo: 'create'?

File: human_annotators_common_errors.py [hunk 2]
```diff
@@ -261,7 +247,6 @@ def create_instruction(file_path: str, json_path: str) -> None:
 
 
 if __name__ == '__main__':
-    URL = 'https://github.com/Ermlab/polish-gec-datasets/tree/main'
-    FILE = 'human_annotators_common_errors_10K.jsonl'
-    file_path, json_path = downloader(URL, FILE)
+    crate_dirs()
```
Comment (pawelkiszczak) [inline human_annotators_common_errors.py]: As above: 'create_dirs'?

**Review by pawelkiszczak — COMMENTED** [review]
Small typos, remainder looks proper and well-arranged.

File: human_annotators_common_errors.py (hunk 1 above)
Comment (IgorTest19) [reply human_annotators_common_errors.py]: It was a typo.

File: human_annotators_common_errors.py (hunk 2 above)
Comment (IgorTest19) [reply human_annotators_common_errors.py]: As above, a typo.

**Pull request merged**